steps/cadence-innovus-power/scripts/main.tcl

```diff
@@ -11,7 +11,9 @@
 #-------------------------------------------------------------------------
 # Generate horizontal stdcell preroutes
 
-sroute -nets {VDD VSS}
+if { $::env(auto-add-M1) } {
+  sroute -nets {VDD VSS}
```

Comment:
Can we just move this line (with the comment header) into both power strategy tcl files? So no parameter.

Reply:
We could, but that would break all of the flows which override the power strategy script without overriding the node.

Reply:
I can do it and make the accompanying changes to the garnet flows, but I'm not sure who else is depending on it. Seems like it might be better to do it in a way that doesn't break existing flows. What do you think?

Reply:
Chimera folks override main.tcl and power strategy, so they don't mind. I think we should do it.